src/lib.rs

```diff
@@ -4,23 +4,124 @@
 use chrono::Local;
 #[cfg(feature = "colored")]
 use colored::*;
-use log::{Level, Log, Metadata, Record, SetLoggerError};
+use log::{Level, LevelFilter, Log, Metadata, Record, SetLoggerError};
+use std::collections::HashMap;
+
+pub struct SimpleLogger {
+    /// The default logging level
+    default_level: Level,
```

Comment:
Shouldn't this be a `LevelFilter` as well, in order to be able to default-disable logging?

Reply:
I find it weird first of being able to disable logger, but you're right.